courses/free-courses-en.md

```diff
@@ -336,6 +336,7 @@
 
 * [Advanced Data Mining with Weka MOOC](https://www.cs.waikato.ac.nz/ml/weka/mooc/advanceddataminingwithweka/)
 * [Applied Data Science with Python](https://cognitiveclass.ai/learn/data-science-with-python) - CognitiveClass.ai
+* [Berkeley's EECS 208: Computational Principles for High-Dimensional Data Analysis](https://pages.github.berkeley.edu/UCB-EECS208/course_site) (University of California, Berkeley)
```

Comment:
Don't invent titles! Use the title given by the resource.
Please add author names where appropriate.
This appears to be time-limited. we don't accept those.

Reply:
Hi, thanks for you advice. I have reverted the commit of Berkeley's EECS 208 course.